connectivity/check/action.go

```diff
@@ -721,18 +721,31 @@ func (a *Action) GetIngressRequirements(p FlowParameters) []filters.FlowSetRequi
 }
 
 // waitForRelay polls the server status from Relay until either it's connected to all the Hubble
-// instances (success) or the context is cancelled (failure).
+// instances, and it started receiving flows (success) or the context is cancelled (failure).
```

Comment:
FWIW The flow count returned by `ServerStatus` is [an aggregation of all the Hubble server answers](https://github.com/cilium/cilium/blob/a1c949c488b23bd46922c9974ccd4eec6475b814/pkg/hubble/relay/observer/server.go#L298-L310), technically no flow is going through Hubble Relay until a `GetFlows` request is made.

Reply:
hmm wonder how i should phrase this 🤔

Reply:
changed it to:

```
// waitForRelay polls the server status from Relay until it indicates that there are no unavailable
// nodes and num_flows is greater than zero (success), or the context is cancelled (failure).
```